test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -3714,10 +3714,10 @@ BOOST_AUTO_TEST_CASE(iterated_keccak256_with_bytes)
 				data[0] = "x";
 				data[1] = "y";
 				data[2] = "z";
-				return keccak256("b", keccak256(data), "a");
+				return keccak256(abi.encodePacked("b", keccak256(data), "a"));
 			}
 		}
-	)";
+	)ABC";
```

Comment:
And this is `R"ABC(` and not `R"YY(` or `R"(` (see question above).

Reply:
What you use between `"` and `(` is irrelevant, as long as you use the same at the end.

Reply:
I meant that some of the changes use `YY` while others `ABC`.